### ISSUE TITLE
Possible to edit the floating terminal window size?

### ISSUE THREAD

**Issue by conspacer** [termux/termux-float#9]
There is a way to change the font size (pinching), but can we change the actual dimensions of the window?
(Fantastic addon to  the superb Termux btw... tx a mil)

**Comment by jdgregson** [comment]
Yes, this is possible. I find that the easiest way (and possibly the correct way) is to long-press on the floating window to enter move mode, and then use another finger to stretch or shrink the window.

**Comment by panigrc** [comment]
@jdgregson unfortunately for us working with a keyboard and mouse, pinching is not possible, is there another way to resize the window ?
Thanx.

**Comment by mentalisttraceur** [comment]
Would love the ability to make the window smaller than the current limit - I can totally understand making the UI not support that (would make it too easy to get it stuck so small that you can't get both fingers on it to zoom it back out), but it would be great to have the option as a configuration setting in the config file.

**Comment by mentalisttraceur** [comment]
Would be happy to take a stab at a PR for this one too, if you're fine with the idea of including the feature.
